ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/efa-pg/provision/cucushift-installer-rehearse-aws-ipi-efa-pg-provision-chain.yaml

```diff
@@ -0,0 +1,24 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-efa-pg-provision
+  steps:
+  - chain: ipi-conf-aws
+  - ref: ipi-conf-aws-custom-region
```

Comment:
What's the purpose of this step?  This may lead to unexpected errors.

Reply:
Because this can only work in us-east-2 region, the placement group needs to be created on AWS in advance, and I only created in us-east-2 region and asked for DPP team to reserve it.

Reply:
Can you create placement group in `us-east-1`? We have an `aws-1-qe` profile that contains `us-east-1` only.

Reply:
Sure, let me create placement group in `us-east-1`. Then we don't need `ref: ipi-conf-aws-custom-region`, right?

Reply:
From my understanding, this feature coverage need some static resources, e.g: `pgcluster` placementGroupName in some specific region. So for me, use `aws-qe-1` cluster profile to replace `aws-qe` can not make the workflow become more common.  To make it can be used more, I am okay with `ipi-conf-aws-custom-region`, also suggest to add more comments about `pgcluster` placementGroupName once `ENABLE_EFA_PG_INSTANCE` is enabled, even I am thinking if we should introduce one more ENV for placementGroupName, so that other users can reuse the step.

Reply:
> ipi-conf-aws-custom-region

People may think _the region can be customized_ , but except install step, all others use LEASE_RESOURCE, they are most likely inconsistent

Reply:
> except install step, all others use LEASE_RESOURCE, they are most likely inconsistent

Yeah, this is indeed a risk, that is why we do not want team mates to override region setting, unfortunately some install workflow already overrode the region setting. 

Okay, let us follow your idea, create placement group in us-east-1, to keep consistent, but one more risk, if in the future, once we add more region in `aws-1-qe` cluster profiles, we probably forget this story.

Reply:
> once we add more regions in `aws-1-qe` cluster profile, we probably forget this story.

No worries, the intention of `aws-1-qe` is to restrict the region list:
* Almost all features come into `us-east-1` first on AWS, e.g. Local Zone
* Some cases require particular region, like this cases

So we won't add additional regions into  `aws-1-qe`.